src/v2/view-builder/views/EnrollFactorEmailView.js

```diff
@@ -3,7 +3,7 @@ import BaseForm from '../internals//BaseForm';
 
 const Body = BaseForm.extend({
   title () {
-    return `Email link sent to (${this.options.appState.get('factorEmail')})`;
+    return `Email link (${this.options.appState.get('factorEmail')})`;
```

Comment:
Maybe change to `Email will be sent to (${this.options.appState.get('factorEmail')})`

Reply:
that's per design https://oktawiki.atlassian.net/wiki/spaces/UX/pages/653230614/Email+Credential